core/variant/callable.h

```diff
@@ -125,6 +125,8 @@ class Callable {
 
 	operator String() const;
 
+	static Callable create(const Variant &p_variant, const StringName &p_method);
```

Comment:
Question about naming: why `create` as opposed to some of the other "creation" verbs used in other places like `new` or `init`?

Reply:
I was thinking between `create()` and `from_variant()`. I think `new()` and `instantiate()` are bad options because of the confusion with `Object.new()` and `PackedScene.instantiate()`.

<details>
<summary>35 matches</summary>

```
35 matches for <method name="(create|new|instantiate|init|from|make)\w*" qualifiers="static">
doc/classes/Basis.xml: 2
 74:  3: 		<method name="from_euler" qualifiers="static">
100:  3: 		<method name="from_scale" qualifiers="static">
doc/classes/Color.xml: 4
144:  3: 		<method name="from_hsv" qualifiers="static">
162:  3: 		<method name="from_ok_hsl" qualifiers="static">
180:  3: 		<method name="from_rgbe9995" qualifiers="static">
187:  3: 		<method name="from_string" qualifiers="static">
doc/classes/DirAccess.xml: 2
237:  3: 		<method name="make_dir_absolute" qualifiers="static">
252:  3: 		<method name="make_dir_recursive_absolute" qualifiers="static">
doc/classes/Image.xml: 2
121:  3: 		<method name="create" qualifiers="static">
131:  3: 		<method name="create_from_data" qualifiers="static">
doc/classes/ImageTexture.xml: 1
 32:  3: 		<method name="create_from_image" qualifiers="static">
doc/classes/MultiplayerAPI.xml: 1
 15:  3: 		<method name="create_default_interface" qualifiers="static">
doc/classes/PhysicsRayQueryParameters2D.xml: 1
 12:  3: 		<method name="create" qualifiers="static">
doc/classes/PhysicsRayQueryParameters3D.xml: 1
 12:  3: 		<method name="create" qualifiers="static">
doc/classes/Projection.xml: 10
 46:  3: 		<method name="create_depth_correction" qualifiers="static">
 53:  3: 		<method name="create_fit_aabb" qualifiers="static">
 60:  3: 		<method name="create_for_hmd" qualifiers="static">
 75:  3: 		<method name="create_frustum" qualifiers="static">
 87:  3: 		<method name="create_frustum_aspect" qualifiers="static">
100:  3: 		<method name="create_light_atlas_rect" qualifiers="static">
107:  3: 		<method name="create_orthogonal" qualifiers="static">
119:  3: 		<method name="create_orthogonal_aspect" qualifiers="static">
131:  3: 		<method name="create_perspective" qualifiers="static">
143:  3: 		<method name="create_perspective_hmd" qualifiers="static">
doc/classes/Quaternion.xml: 1
 93:  3: 		<method name="from_euler" qualifiers="static">
doc/classes/Vector2.xml: 1
201:  3: 		<method name="from_angle" qualifiers="static">
modules/gltf/doc_classes/GLTFCamera.xml: 2
 15:  3: 		<method name="from_dictionary" qualifiers="static">
 22:  3: 		<method name="from_node" qualifiers="static">
modules/gltf/doc_classes/GLTFLight.xml: 2
 14:  3: 		<method name="from_dictionary" qualifiers="static">
 21:  3: 		<method name="from_node" qualifiers="static">
modules/gltf/doc_classes/GLTFPhysicsBody.xml: 2
 14:  3: 		<method name="from_dictionary" qualifiers="static">
 21:  3: 		<method name="from_node" qualifiers="static">
modules/gltf/doc_classes/GLTFPhysicsShape.xml: 2
 15:  3: 		<method name="from_dictionary" qualifiers="static">
 22:  3: 		<method name="from_node" qualifiers="static">
modules/regex/doc_classes/RegEx.xml: 1
 65:  3: 		<method name="create_from_string" qualifiers="static">
```

</details>

Reply:
I think `create` is fine. It doesn't overlap with anything that could potentially be added for Callable. Combined with the parameters it can be thought as "create a Callable for this variant's method with the given name).

Granted, it's a bit misleading because for Objects it doesn't really create them, per say, but you know.

Actually the second parameter could be called `method_name`. There's a lot of misleading Callable parameter names and I wouldn't want this to be one of them, too.

Reply:
If anything, I'd say `from_string_name` would be more appropriate than `from_variant`, but in any case, I think you've made a convincing argument that `create` is a good choice after all.

Reply:
> Granted, it's a bit misleading because for Objects it doesn't really create them, per say, but you know

This method actually creates a `Callable` as a value _representing_ method/function. Yes, we have differences between standard and custom callables (the latter can hold a reference), but I think that this has nothing to do with the creation of the callable itself.

> Actually the second parameter could be called `method_name`.

I'd prefer to keep the parameter names consistent with the constructor.